### DISCUSSION TITLE
Saving drawingView stream on Android version lower than 10

### DISCUSSION THREAD

**Discussion by petarkrastev** [CommunityToolkit/Maui#2271] [Q&A]
When saving DrawingView stream (DrawingView.GetImageStream()) on Android versions lower than 10 it is losing the lines colors and is saving everything as black, no matter what your colors were

**Comment by ghost** [comment]
Just for reference, similar bug was fixed - with [transparent color](https://github.com/CommunityToolkit/Maui/issues/1246). @petarkrastev is this still relevant?
